2023-10-15/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+)
+
+func concatenate(a, b string) string {
```

Comment:
since both of the variables are of the same type its enough to declare the last variable type. would they have different types we would have to declare them at the end of the list of parameters.

Reply:
of said parameters... so like:
`a, b, c string, d, e int`